sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -562,216 +562,186 @@
-                    "custom": "",
-                    "customPrivate": false
-                  },
-                  "conditional": {
-                    "show": "true",
-                    "when": "regulatingBody",
-                    "eq": "other"
-                  },
-                  "type": "textfield",
-                  "labelPosition": "top",
-                  "inputFormat": "plain",
-                  "tags": [],
-                  "properties": {},
-                  "lockKey": true
-                }
-              ],
-              "width": 6,
-              "offset": 0,
-              "push": 0,
-              "pull": 0
-            }
-          ],
-          "type": "columns",
-          "hideLabel": true,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "allowCalculateOverride": false,
+          "validateOn": "change",
+          "validate": {
+            "required": true,
+            "onlyAvailableItems": true,
+            "customMessage": "",
+            "custom": "",
+            "customPrivate": false,
+            "json": "",
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "unique": false,
+          "errorLabel": "",
+          "errors": "",
+          "key": "regulatingBody",
           "tags": [],
+          "properties": {},
           "conditional": {
             "show": "",
             "when": null,
-            "eq": ""
+            "eq": "",
+            "json": ""
           },
-          "properties": {}
+          "customConditional": "",
+          "logic": [],
+          "attributes": {
+            "data-cy": "regulatingBody"
+          },
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "select",
+          "indexeddb": {
+            "filter": {}
+          },
+          "selectFields": "",
+          "searchField": "",
+          "searchDebounce": 0.3,
+          "minSearch": 0,
+          "filter": "",
+          "redrawOn": "",
+          "input": true,
+          "searchThreshold": 0.3,
+          "prefix": "",
+          "suffix": "",
+          "dataGridLabel": false,
+          "showCharCount": false,
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "addons": [],
+          "lazyLoad": true,
+          "authenticate": false,
+          "ignoreCache": false,
+          "fuseOptions": {
+            "include": "score",
+            "threshold": 0.3
+          },
+          "id": "e5eqny",
+          "defaultValue": ""
+        },
+        {
+          "autofocus": false,
+          "input": true,
+          "tableView": true,
+          "inputType": "text",
+          "inputMask": "",
+          "label": "Other institution regulating body",
```

Comment:
The last PR raised the question (answered by @hellolynn-tbtb) about using `Regulating` vs `Regulatory` and apparently the expected way would be `Regulatory` and per the wiki below.
The institution profile is right but apparently the programs are not.
@JasonCTang does it make sense? Should we adjust the programs?

![image](https://github.com/bcgov/SIMS/assets/61259237/0eda32e3-0227-47f5-aaec-53aaa4282692)
Source: https://github.com/bcgov/SIMS/wiki/Acronyms-&-Terminology#word--spelling-usage